PLAY-ro.md

```diff
@@ -8,12 +8,12 @@ NetGuard
 
 Short description
 -----------------
-Netguard, primul firewall de Android cu sursa deschisa ce nu necesita root 
+Netguard, primul firewall de Android gratis si cu sursa deschisa ce nu necesita root 
```

Comment:
The long description is maximum 80 characters.

Would this be okay: '*Netguard, primul firewall gratis si cu sursa deschisa ce nu necesita root*'.